spec/factories/in_progress_forms/hca_forms.rb

```diff
@@ -8,7 +8,7 @@
       {
         savedAt: 1_595_954_803_670,
         version: 6,
-        return_url: '/form-url/review-and-submit'
+        returnUrl: '/form-url/review-and-submit'
```

Comment:
I think that this should remain unchanged.  HCA will still use inflection headers for now, so it would be `return_url` in the database.